src/ucsi/lpm/get_connector_status.rs

```diff
@@ -286,10 +289,10 @@ impl TryFrom<u8> for ConnectorPartnerFlags {
 
     fn try_from(raw: u8) -> Result<Self, Self::Error> {
         match raw {
-            0x0 => Ok(ConnectorPartnerFlags::Usb),
```

Comment:
I might be missing something, but this looks odd to me - the change from sequential to powers-of-two implies to me that these are expected to be bitwise-or'd together as a sort of 'flags' field to express that more than one of these can apply simultaneously, but this parser will only work if one flag is set? Is that intentional?  I'm not super familiar with the details of USB so I'm not sure if it would make sense to have something like (AltMode | Usb4Gen4) as a value here?